alfresco-search/src/main/java/org/alfresco/solr/SolrInformationServer.java

```diff
@@ -1674,6 +1657,88 @@ public void putAclTransactionState(UpdateRequestProcessor processor, SolrQueryRe
         }
     }
 
+    public void capIndex(long dbid) throws IOException
+    {
+        SolrQueryRequest request = getLocalSolrQueryRequest();
+        UpdateRequestProcessor processor = this.core.getUpdateProcessingChain(null).createProcessor(request, new SolrQueryResponse());
+        AddUpdateCommand cmd = new AddUpdateCommand(request);
+        cmd.overwrite = true;
+        SolrInputDocument input = new SolrInputDocument();
+        input.addField(FIELD_SOLR4_ID, "TRACKER!STATE!CAP");
+        input.addField(FIELD_VERSION, 0);
+        input.addField(FIELD_DBID, -dbid); //Making this negative to ensure it is never confused with node DBID
```

Comment:
Could confuse unbounded lower range if we add range support for DBID ...

Reply:
Can you elaborate on this?